mito-ai/mito_ai/handlers.py

```diff
@@ -73,8 +73,8 @@ def log(self) -> logging.Logger:
         return get_logger()
 
     @tornado.web.authenticated
-    def head(self) -> None:
-        """Handle a HEAD request for the websocket."""
+    def get_service_status(self) -> None:
```

Comment:
On dev this old `head` function was getting triggered, but in this new branch the `get_service_status` function doesn't seem to get called. Is that an intentional change? If so, can we document what's going on here.

Reply:
Good catch! `get_service_status` was called initially and then through refactors I lost that call. Ultimately, I just moved what the function does inside of `get` function now